internal/service/streamaccountdetails/data_source_schema.go

```diff
@@ -0,0 +1,73 @@
+package streamaccountdetails
+
+import (
+	"context"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+)
+
+func DataSourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"aws_account_id": schema.StringAttribute{
+				Computed:            true,
+				Description:         "The AWS Account ID.",
+				MarkdownDescription: "The AWS Account ID.",
```

Comment:
to reduce verbosity you can define only `MarkdownDescription` and make use of `UpdateSchemaDescription` to set the value of `Description` ([example](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/service/streamprocessor/resource.go#L43))